tests/extensions/vb/Microsoft.DotNet.UpgradeAssistant.Extensions.VisualBasic.Tests/MyTypeTargetFrameworkSelectorFilterTests.cs

```diff
@@ -35,7 +39,7 @@ public void ProcessTests(string myType, bool update)
 
             // Assert
             var expectedTimes = update ? Times.Once() : Times.Never();
-            state.Verify(s => s.TryUpdate(TargetFrameworkMoniker.Net50_Windows), expectedTimes);
+            state.Verify(s => s.TryUpdate(TargetFrameworkMoniker.Net60), expectedTimes);
```

Comment:
Shouldn't we expect try update to be called with a Windows-specific TFM?

Reply:
ah yes, w/ the latest change this would also be needing an update - I just accepted the suggestion inline 😅 fixing now.